### PR TITLE
issue 5433 fix

### LINKED ISSUE #5433: [$20] Update Forums settings preferences
**Current behavior:** 
When user is on the profile preferences page and clicks on FORUMS, the user is taken to the legacy forums at https://apps.topcoder.com/forums/?module=Settings

**Expected behavior:**
When user is on the profile preferences page and clicks on FORUMS, the user is taken to the Vanilla Forums preferences settings page at https://discussions.topcoder.com/profile/preferences

**Steps to reproduce:**
1) Log in to Topcoder.com
2) go to https://www.topcoder.com/challenges?tracks[DS]=true&tracks[Des]=true&tracks[Dev]=true&tracks[QA]=true&types[]=CH&types[]=F2F&types[]=TSK
3) then go to your profile settings https://www.topcoder.com/settings/profile
4) then go to https://www.topcoder.com/settings/preferences 
5) click on FORUMS on the leftside
6) you are taken to the Legacy JIVE forums preference settings

image

### PR DESCRIPTION
**Pull request by rashmi73**
#5433

### CODE REVIEW

File: config/production.js [hunk 1]
```diff
@@ -31,6 +31,7 @@ module.exports = {
     HOME: '/my-dashboard',
     COMMUNITY: 'https://community.topcoder.com',
     FORUMS: 'https://apps.topcoder.com/forums',
+    DISCUSSION_FORUM: 'https://discussions.topcoder.com',
```
Comment (luizrrodrigues) [inline config/production.js:34]: We already have a link to discuss forum in config: `FORUMS_VANILLA`

File: config/default.js [hunk 2]
```diff
@@ -111,6 +111,7 @@ module.exports = {
     BLOG_FEED: 'https://www.topcoder.com/blog/feed/',
     COMMUNITY: 'https://community.topcoder-dev.com',
     FORUMS: 'https://apps.topcoder-dev.com/forums',
+    DISCUSSION_FORUM: 'https://discussions.topcoder.com',
```
Comment (luizrrodrigues) [inline config/default.js:114]: We already have a link to discuss forum in config: `FORUMS_VANILLA`

File: src/shared/components/Settings/Preferences/index.jsx [hunk 3]
```diff
@@ -88,7 +88,7 @@ export default class Preferences extends React.Component {
       case 'e-mail':
         return <NewsletterPreferencesContainer email={email} />;
       case 'forum':
-        return (window.location.href = `${config.URL.FORUMS}/?module=Settings`) && <LoadingIndicator />;
+        return (window.location.href = `${config.URL.DISCUSSION_FORUM}/profile/preferences`) && <LoadingIndicator />;
```
Comment (luizrrodrigues) [inline src/shared/components/Settings/Preferences/index.jsx:91]: Please use `FORUMS_VANILLA`

**Review by luizrrodrigues — CHANGES_REQUESTED** [review]

File: src/shared/components/Settings/Preferences/index.jsx (hunk 3 above)
Comment (rashmi73) [reply src/shared/components/Settings/Preferences/index.jsx:91]: @luizrrodrigues ok updating PR in sometime

File: config/production.js (hunk 1 above)
Comment (rashmi73) [reply config/production.js:34]: @luizrrodrigues ok updating PR in sometime

**Review by luizrrodrigues — APPROVED** [review]

**Pull request merged**